etna/core/tests/test_decorators.py

```diff
@@ -0,0 +1,71 @@
+from django.test import TestCase, override_settings
+from django.urls import reverse_lazy
+
+from wagtail.tests.utils import WagtailTestUtils
+
+import responses
+
+from ...ciim.tests.factories import create_response
+
+CONDITIONALLY_PROTECTED_URLS = (
+    reverse_lazy("search-catalogue"),
+    reverse_lazy("image-browse", kwargs={"iaid": "C140"}),
+    reverse_lazy("details-page-machine-readable", kwargs={"iaid": "C140"}),
+)
```

Comment:
Generic - Is there any time its too much to include in the test. Is it fine to have some than all - for example urls in this list? Of course, I imagine its really helpful in getting some testcases here than it not being there at all. Thanks for that.

Reply:
I don't have any solid rules for where to put URLs, only that using `reverse_lazy` is generally more flexible than `reverse`, because it doesn't require all of the URLs to have been registered yet. These could just as well live on the test case class, or elsewhere. In this case, I felt like having them as a global better highlighted their importance to the rest of the module.

Reply:
Going with 'not having any solid rules'.